### ISSUE TITLE
Images are blank

### ISSUE THREAD

**Issue by daltonrooney** [divamgupta/diffusionbee-stable-diffusion-ui#26]
I tried a text -> image prompt using the standard image settings, the image generated but is just a solid black PNG.  Let me know where to find the logs and I will upload them. 14" Macbook Pro M1 with 16GB of RAM.

**Comment by BrianGilbert** [comment]
I get random black PNG's as well, on prompts that don't have any hint of NSFW

**Comment by daltonrooney** [comment]
It’s working for me if I reduce the images to 256px. Maybe a memory issue? I knew I should have gone for 64GB! 😭

**Comment by drewlustro** [comment]
Happens every other generation for me. Just try again!

**Comment by ee0pdt** [comment]
@daltonrooney  I'm on 14" Macbook Pro M1 Max with 16GB of RAM and it works fine at 512 x 512. Doubt it's a memory issue. 

Might be worth quitting all other apps and/or restarting your machine to see if it helps

**Comment by isbkch** [comment]
MBP 16" M1 Max with 32 GB of RAM and I still get black PNGs every once in a while

**Comment by harryshawk** [comment]
I got a black image today as well.

**Comment by victorbasu94** [comment]
I'm getting black images now too. NSFW filter isn't on - I've validated that

**Comment by elbowdonkey** [comment]
When I've seen this happen, it's usually when DiffusionBee is in the background and I'm doing something somewhat CPU heavy. I've not yet tried to tease apart why it happens, but that part at least feels consistent.

**Comment by hellbox** [comment]
I've also seen this behavior at what seems like random times (2021 16" M1 MBP / 16gb). If I rerun the prompt it always works the second time, and the inputs were relatively benign so I doubt NSFW filter.

**Comment by elbowdonkey** [comment]
I just found [this issue](https://github.com/pytorch/pytorch/issues/84364) for another M1 capable fork of stable diffusion - might explain the black images.

@hellbox fancy meeting you here.

**Comment by Birch-san** [comment]
no, the "black images from k-diffusion" bug is different (it happened 100% of the time, not sporadically).

sporadic black images is something I'm aware of (which I noticed on PLMS sampler, at high numbers of steps):
https://github.com/lstein/stable-diffusion/issues/517#issuecomment-1247957779  
but haven't investigated, because the problem never happens to me any more. I use low step counts (e.g. 8), and k-diffusion samplers.

**Comment by masaldana2** [comment]
<img width="990" alt="Screenshot 2022-09-22 at 9 05 45 PM" src="https://user-images.githubusercontent.com/7034219/191880100-e91ba58f-b7ca-4916-ab86-894974c7d854.png">

this I all I get every time

**Comment by ProStrike-hub** [comment]
I also have the same problem every time, even when I decrease steps to 10... So I doubt this is because of my computer's performance.

Couldn't locate this NSFW filter you're talking about. 

Anyone has a fix please?
